src/compiler/commandLineParser.ts

```diff
@@ -1341,7 +1341,12 @@ namespace ts {
     /**
      * Reads the config file, reports errors if any and exits if the config file cannot be found
      */
-    export function getParsedCommandLineOfConfigFile(configFileName: string, optionsToExtend: CompilerOptions, host: ParseConfigFileHost): ParsedCommandLine | undefined {
+    export function getParsedCommandLineOfConfigFile(
+        configFileName: string,
+        optionsToExtend: CompilerOptions,
+        host: ParseConfigFileHost,
+        /*@internal*/ extendedConfigCache?: Map<ExtendedConfigCacheEntry>

```

Comment:
unfortunately that makes it impossible for API users to benefit from this caching.